src/components/NotificationsList.tsx

```diff
@@ -24,60 +27,98 @@ const NotificationsList: React.FunctionComponent<{
             description,
             sprint,
           } = notification;
-          return (
-            <div key={i} className="border-b-2 border-dotted border-gray-200 pb-4 mb-4">
-              <div className="text-sm">
+          let isAcknowledgement = true;
+          let notificationDescription;
+          switch (notification.type) {
```

Comment:
I see that the commit message says 'corresponding change to backend' to switch to this from `notification['@type']`. 👍  noted that this works when BE change is out

Reply:
It's out. I also updated the other WIP branch with it.

Reply:
cool thanks!